src/base/print/printRxnFormula.m

```diff
@@ -11,14 +11,17 @@
 % INPUTS:
 %    model:             COBRA model structure
 %
-% OPTIONAL INPUTS:
+% OPTIONAL INPUTS
 %    rxnAbbrList:       Abbrs of reactions whose formulas are to be printed
+%
+% OPTIONAL INPUTS as 'ParameterName',Value pairs:
```

Comment:
True, But essentially, it woudl just be another Option wouldn't it? So a third case in addition to 
INPUT
OPTIONAL INPUT
OUTPUT
OPTIONAL OUTPUT
and new e.g.:
OPTIONAL PARAMETER VALUE PAIRS